assets/src/components/d3/createProgressBar.js

```diff
@@ -47,8 +47,14 @@ function createProgressBar ({ data, width, height, el, tip }) {
     .style('outline-style', 'solid')
     .style('outline-width', '1px')
 
+  // bar.append('text')
```

Comment:
Commented out the text because it was messing up the alignment of the tooltip, as the `<text>` is not bounded to the `<rect>`. Is it worth keeping the text here? In the previous implementation, I believe the first 3 or 4 characters of the text were sliced and displayed. I'm wondering if there's a more UI friendly way to implement this.

One idea that was floated around in our office was highlighting the progress bar rectangle when user hovers over corresponding table row, and vice versa. Thoughts?

Reply:
> Commented out the text because it was messing up the alignment of the tooltip, as the <text> is not bounded to the <rect>. Is it worth keeping the text here? In the previous implementation, I believe the first 3 or 4 characters of the text were sliced and displayed.
It was one of the requirements that was asked, we could we CSS overflow:ellipsis. So we still need that. But we don't want to add test with bar % is too small.

Reply:
> One idea that was floated around in our office was highlighting the progress bar rectangle when user hovers over corresponding table row, and vice versa. Thoughts?

It is interesting idea, but for assignments with 0% will end up in table view but not in the progress view. So we might need to consider that case when implementing.

Reply:
> It was one of the requirements that was asked, we could we CSS overflow:ellipsis. So we still need that. But we don't want to add text when bar % is too small. I think having the text on progress bar in there will let users hover over it for more info.

Okay I've implemented this here: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/447/commits/36d7b2a24553000097c819992240a94dbae3a958